.pre-commit-config.yaml

```
@@ -0,0 +1,11 @@
+---
+repos:
+  - repo: git://github.com/pre-commit/pre-commit-hooks
```

```suggestion
  - repo: https://github.com/pre-commit/pre-commit-hooks
```
because of https://github.com/thoth-station/thoth-application/issues/2111